kind.md

```diff
@@ -2,3 +2,40 @@
 
 ### Setup ovn-kubernetes on KIND
 
+You can follow [this guide](https://github.com/ovn-org/ovn-kubernetes/blob/master/docs/kind.md#ovn-kubernetes-kind-setup) either with docker or podman
+
+### Load local image
+You can load any local docker images in kind without pushing them online
```

Comment:
pretty cool, I didn't know that :)

Reply:
all credits goes to @OlivierCazade :1st_place_medal: